README.md

```diff
@@ -108,6 +109,8 @@ more CircleCI resources than are necessary.
 
 Command that installs your application's node modules and Cypress dependencies.
 
+> ⚠️ Note: this command is only necessary if you plan to execute the `run` command later. Especially if you run the tests on multiple machines in parallel.
```

Comment:
I think we should specify when it should run instead of just saying later:

`Note: this command is only necessary if you plan to execute the `run-tests` command in a separate `job`

Reply:
Makes sense. I had pulled it in from the old version. I like this change.